extra/utils/permissions.js

```diff
@@ -0,0 +1,26 @@
+const permissions = {
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    delete: [],
+    }
+    }
+    function haspermission(modulename,role,permissiontype)
```

Comment:
Please use camel casing for variable and function names

Reply:
Done